.travis.yml

```diff
@@ -1,11 +1,19 @@
-sudo: false
-dist: trusty
-language: python
-python:
-  - 3.6
+sudo: required
+language: c
+env:
+    matrix:
+        - DOCKER_IMAGE=nmstate/centos7-nmstate-dev
+        - DOCKER_IMAGE=nmstate/fedora-nmstate-dev
+addons:
+  apt:
+    packages:
+    - realpath
 
-install:
-  - pip install tox
+services:
+    - docker
+
+before_install:
+    - docker pull $DOCKER_IMAGE
 
 script:
-  - tox -e flake8,pylint,check-py27,check-py36,coveralls
+  - sudo env DOCKER_IMAGE=$DOCKER_IMAGE ./automation/run-integration-tests.sh
```

Comment:
the script should be renamet to just run-tests, since it is all tests now and not just integration tests